pyproblems/prime_factor_products.py

```diff
@@ -0,0 +1,18 @@
+'''This program is to find out the product of the prime factors of a given number'''
+from pyproblems.utility import is_int
+from pyproblems.prime import is_prime
+
+def product_prime(num1):
+    '''This function returns the product of the numbers which are prime factors
+    of a given number'''
+    if not is_int(num1):
+        raise TypeError(f'unsupported format, {type(num1)} is in the place of"integer"')
+    if num1 <= 0:
+        raise ValueError("pass an integer greater than 0")
+    product = 1
+    for i in range(2, num1+1):
+        if num1 % i == 0:
+            if is_prime(i):
```

Comment:
Why there are two if statements. it can be like this num1%i == 0 and is_prime(i)

Reply:
Okay sir. I will fix that.

Reply:
Sir, The code is not giving the expected output if I try it the way you mentioned.

Reply:
Hi sir, I tried again. Made some parallax error. Now it is working fine.